tests/x11regressions/tracker/tracker_searchall.pm

```diff
@@ -18,15 +18,6 @@ sub run() {
     sleep 2;
     wait_idle;
     assert_screen 'tracker-needle-launched';
-    send_key "tab";
```

Comment:
@coolo I see, I will add 

    if (get_var("SP2ORLATER"))

to make it works on both SP1 and SP2